pkg/cmd/tests/tests.go

```diff
@@ -46,6 +50,14 @@ var Suites = ginkgotest.TestSuites{
 		LabelFilter:        fmt.Sprintf("%s", framework.SerialLabelName),
 		DefaultParallelism: 1,
 	},
+	{
+		Name: "scylla-operator/conformance/multi-datacenter-parallel",
```

Comment:
Why would we have a dedicated suite for something that's hardwired into the conformance?

Reply:
How is it hardwired into the conformance? I don't understand. The intended meaning behind it is `conformance suite for multi-datacenter infrastructure`. How'd you do it otherwise?

Reply:
scratch that, I missed the `!` two lines above